debian/control

```diff
@@ -1200,6 +1201,32 @@ Description: Python 3 utility libraries for Ceph CLI
  This package contains types and routines for Python 3 used by the
  Ceph CLI as well as the RESTful interface.
 
+Package: python-ceph-common
+Architecture: all
+Section: python
+Depends: ${misc:Depends},
+         ${python:Depends},
+Description: Python 2 utility libraries for Ceph
+ Ceph is a massively scalable, open-source, distributed
+ storage system that runs on commodity hardware and delivers object,
+ block and file system storage.
+ .
+ This package contains data structures, classes and functions used by Ceph.
+ It also contains utilities used for the SSH orchestrator.
```

Comment:
maybe too much specific? what about 
"It also contains utilities used in manager modules"

Reply:
fixed